src/pages/my-courses/MyCourses.tsx

```diff
@@ -31,22 +40,20 @@ import { styles } from '~/pages/my-courses/MyCourses.styles'
 const MyCourses = () => {
   const { t } = useTranslation()
   const breakpoints = useBreakpoints()
-  const { page, handleChangePage } = usePagination()
   const { openDialog } = useConfirm()
   const { setAlert } = useSnackBarContext()
-  const searchTitle = useRef<string>('')
   const { sort, onRequestSort } = useSort({ initialSort })
   const itemsPerPage = getScreenBasedLimit(breakpoints, courseItemsLoadLimit)
 
+  const { filters, activeFilterCount, searchParams, filterQueryActions } =
+    useFilterQuery({
+      defaultFilters: defaultFilters,
+      countActiveFilters: countActiveCourseFilters
```

Comment:
```suggestion
      defaultFilters,
      countActiveFilters: countActiveCourseFilters
```

Reply:
This lines are outdated, now it looks like this:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/104922087/9aecac23-1592-45ac-a81d-40701de07e83)